vitest.setup.ts

```diff
@@ -0,0 +1,9 @@
+import * as matchers from '@testing-library/jest-dom/matchers'
+import { cleanup } from '@testing-library/react'
```

Comment:
So most of the actual test code is actually going to be using the React Testing Library, which is what I believe we've settled on as an organisation.